app/background/wallet/service.js

```diff
@@ -738,6 +759,17 @@ class WalletService {
     },
   );
 
+  signMessage = async (txJSON, passphrase) => {
```

Comment:
I think `signTX` is a better name for this since its not an arbitrary message the way `signMessageWithName` is. Although this function may be moved anyway if you refactor the wallet proxy thing.